includes/rest-api/class-sensei-rest-api-lesson-quiz-controller.php

```diff
@@ -182,29 +182,40 @@ public function save_quiz( WP_REST_Request $request ) {
 		$json_params = $request->get_json_params();
 
 		$quiz_id = wp_insert_post(
-			[
+			array(
 				'ID'           => $quiz_id,
 				'post_content' => '',
 				'post_status'  => $lesson->post_status,
 				'post_title'   => $lesson->post_title,
 				'post_type'    => 'quiz',
 				'post_parent'  => $lesson->ID,
 				'meta_input'   => $this->get_quiz_meta( $json_params, $lesson ),
-			]
+			)
 		);
 
 		if ( is_wp_error( $quiz_id ) ) {
 			return $quiz_id;
 		}
 
 		if ( $is_new ) {
-			update_post_meta( $lesson->ID, '_lesson_quiz', $quiz_id );
-			wp_set_post_terms( $quiz_id, [ 'multiple-choice' ], 'quiz-type' );
+			/**
+			 * Fires after a quiz is created via the REST API.
+			 *
+			 * @since $$next-version$$
+			 *
+			 * @hook sensei_rest_api_lesson_quiz_created
+			 *
+			 * @param {int} $quiz_id   Quiz post ID.
+			 * @param {int} $lesson_id Course post ID.
+			 */
+			do_action( 'sensei_rest_api_lesson_quiz_created', (int) $quiz_id, (int) $lesson->ID );
```

Comment:
Here is the only meaningful change, the rest is the code style fix.

Here we fire an action on the creation of a new quiz within a block editor.